LoopKit/InsulinKit/HKQuantitySample+InsulinKit.swift

```diff
@@ -110,6 +116,10 @@ extension HKQuantitySample {
         return metadata?[MetadataKeyProgrammedTempBasalRate] as? HKQuantity
     }
 
+    var duringSuspend: Bool? {
```

Comment:
[nit] Consider making this non-optional (see above)

Reply:
No longer necessary. Removing `duringSuspend` per other PR feedback.

Reply:
Actually, necessary here, but not in DoseEntry. Made non-optional per feedback.